UnityProject/Assets/Scripts/Closets/ClosetControl.cs

```diff
@@ -195,12 +194,15 @@ private void SetItems(bool open)
 
         private void SetItemsAliveState(bool on)
         {
-            if (!on)
-                heldItems = matrix.Get<ObjectBehaviour>(registerTile.Position, ObjectType.Item);
+			if (!on)

```

Comment:
indentation is a little crappy here, but will do a automated formatting before we go to QA for 0.2.2 anyway

Reply:
Its because I use tabs, not spaces, and github shows tabs as 8 spaces. (MADNESS!). It will look as it should in your IDE :)

Reply:
Yeah, I think so too...
You're not someone to screw indentation up on purpose :P